functional_tests/test_dcp_reshard.py

```diff
@@ -47,15 +47,12 @@ def test_dcp_reshard_sync_gateway_goes_down(cluster, conf):
         futures[executor.submit(traun.add_docs, 2000, bulk=True)] = "traun"
 
         for future in concurrent.futures.as_completed(futures):
-            try:
-                tag = futures[future]
-                if tag == "sg_down":
-                    # Assert takedown was successful
-                    shutdown_status = future.result()
-                    assert shutdown_status == 0
-                print("{} Completed:".format(tag))
-            except Exception as e:
-                print("Exception thrown while adding docs: {}".format(e))
+            tag = futures[future]
+            if tag == "sg_down":
+                # Assert takedown was successful
+                shutdown_status = future.result()
+                assert shutdown_status == 0
+            print("{} Completed:".format(tag))
```

Comment:
The command never gets printed if the assert fails. This can be moved up before assert. Also if this can be changed to logging module.